client/web/compose/src/components/Public/Record/BulkEdit/index.vue

```diff
@@ -77,13 +77,13 @@
 
           <div>
             <b-button
-              variant="link"
+              variant="light"
               rounded
-              class="text-primary"
```

Comment:
Previously it wasn't obvious since it was a link, but this button doesn't have the correct margin. You can add gap to the footer